### DISCUSSION TITLE
Wallet FAQ

### DISCUSSION THREAD

**Discussion by heycorwin** [near/near-wallet#1226] [Ideas]
Let's begin aggregating what we believe to be good questions to place in an FAQ for the wallet. These should be things that appear in the UI but may be technical and might benefit from further explanation than what the limited space within a UI can house.

**Comment by heycorwin** [comment]
Q: Why is 35 NEAR required to enable 2FA for my account?

**Reply by toddpierpoint** [reply]
Doesn't it take 36 NEAR or is the UI incorrect?
<img width="424" alt="Screen Shot 2020-11-25 at 11 55 57 AM" src="https://user-images.githubusercontent.com/73570885/100265241-9674fc80-2f15-11eb-95ce-70cd288fe397.png">

**Reply by heycorwin** [reply]
Sorry for any confusion @toddpierpoint, I kind of just spit out a number that I've been carrying around in my head for the sake of giving an example. The UI should be correct in it's representation of the required amount.

**Comment by heycorwin** [comment]
Q: What is a lockup contract? How is it and the balances it contains different from my regular account? How are they related?

**Comment by heycorwin** [comment]
Q: What are security best-practices?

**Reply by vgrichina** [reply]
Best practice would be to use your web wallet in mobile browser or use Ledger if desktop access is needed. Otherwise pretty big exposure to malicious browser extensions / other common malware. 2FA helps with this, but only if not otherwise accessible from same machine (e.g. if same malware can read email from app it’s not very effective).

**Comment by heycorwin** [comment]
Q: Why do I need NEAR tokens to create an account?

**Reply by kcole16** [reply]
For a crypto-familiar user
- Accounts on NEAR are similar to smart contract-based accounts on other blockchains (e.g. Ethereum). These accounts are powerful, they can: have a human-readable name, create keys with specific app-based permissions, and add or remove keys to improve account security. In order to create these accounts on the NEAR blockchain, a transaction costing 1 NEAR is required.

For a non-crypto user
- Creating a NEAR account requires reserving storage space on the NEAR blockchain. The cost of this space is about 1 NEAR.

**Comment by heycorwin** [comment]
Q: Where can I buy NEAR to fund my account?

**Reply by kcole16** [reply]
You can acquire NEAR in the following ways:
- With a credit/debit card on Moonpay (non-US only)
- By using an exchange: Binance, Huobi, OkEx
- By buying wNEAR on Uniswap on Ethereum, then transferring it to your account using the rainbow bridge (available Q1)
